include/AIView.h

```diff
@@ -7,9 +7,9 @@
 
 class AIView: public View {
 	private:        
-		
+		int gameState;
```

Comment:
Why is gameState a member variable if it is going to be dynamic throughout the game?